Directory.Build.props

```diff
@@ -347,6 +347,7 @@
     <DefineConstants Condition="'$(TargetFramework)' == 'netstandard2.1'">$(DefineConstants),netstandard21</DefineConstants>
     <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp'">$(DefineConstants),netcoreapp</DefineConstants>
     <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp2.0'">$(DefineConstants),netcoreapp20</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp3.0'">$(DefineConstants),netcoreapp30</DefineConstants>
```

Comment:
nit: we should also add netcoreapp5.0 here.

Reply:
Actually what's the reasons that we don't use the pre-defined symbols? ` i.e. `NETCOREAPP, NETCOREAPP1_0, NETCOREAPP1_1, NETCOREAPP2_0, NETCOREAPP2_1, NETCOREAPP2_2`

https://docs.microsoft.com/en-us/dotnet/core/tutorials/libraries

Reply:
This predated an MSBuild based SDK :).  Agreed we should standardize on those and let the SDK define them for us.  Care to open an issue for that?

Reply:
https://github.com/dotnet/corefx/issues/40484